client/src/helpers/hooks/useAddressBalance.ts

```diff
@@ -55,6 +55,13 @@ export function useAddressBalance(
                         setBalance(totalBalance);
                         setAvailableBalance(null);
                     }
+                } else {
```

Comment:
Looks like this else is never reached but rather the `else if (isMounted)` is executed